doc/init.rst

```diff
@@ -0,0 +1,11 @@
+:orphan:
+
+Create dummy project
+--------------------
+
+After installing Universum via ``pip3.7 install -U universum`` or simply from a downloaded project root directory
```

Comment:
"downloaded project root directory" might be a bit confusing, could you please rephrase?